### DISCUSSION TITLE
latex模版目录没有摘要什么的怎么加嘞？？？

### DISCUSSION THREAD

**Discussion by wangduxing** [sjtug/SJTUThesis#1073] [提问] [answered]
latex模版目录没有摘要什么的怎么加嘞？？？
latex↓
<img width="415" alt="04735507ba985b246f738e06a0b3a77" src="https://github.com/user-attachments/assets/5f9750a6-317f-4d9a-b467-45be2586a932">
word↓
<img width="595" alt="6fa25193f7ca9df2f2c04edffa65ec5" src="https://github.com/user-attachments/assets/730fae3e-e2f6-4280-b79b-e9360122efb7">

**Comment by NemoYuan2008** [comment, marked as answer]
将代码中 
```latex
\begin{abstract*}[zh]
  ...
\end{abstract*}
```
改为
```latex
\begin{abstract}[zh]
  ...
\end{abstract}
```
即可。英文摘要同理。

**Reply by wangduxing** [reply]
wow，谢谢啦~~~亲测有效
